project/Build.scala

```diff
@@ -1036,7 +1062,7 @@ object Build extends sbt.Build {
   lazy val partest: Project = Project(
       id = "partest",
       base = file("partest"),
-      settings = commonSettings ++ Seq(
+      settings = commonSettings ++ fatalWarningsSettings ++ Seq(
```

Comment:
This is the cause of the build failure. partest sources in the tools, so it builds with warnings, unfortunately.